egov-persister/vehicle-persister.yaml

```diff
@@ -149,6 +149,10 @@ serviceMaps:
   
             - jsonPath: $.vehicleTrip.auditDetails.lastModifiedTime
             
+	    - jsonPath: $.vehicleTrip.fstpEntryTime
```

Comment:
alignment of the fstpEntryTime is wrong, due to which persister will fail. please correct this.